pkg/conversions/virtualnode_types_conversion_test.go

```diff
@@ -0,0 +1,60 @@
+package conversions
+
+import (
+	appmesh "github.com/aws/aws-app-mesh-controller-for-k8s/apis/appmesh/v1beta2"
+	"github.com/aws/aws-sdk-go/aws"
+	appmeshsdk "github.com/aws/aws-sdk-go/service/appmesh"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/conversion"
+	"testing"
+)
+
+func TestConvert_CRD_TLSValidationContextACMTrust_To_SDK_TLSValidationContextACMTrust(t *testing.T) {
```

Comment:
> That's quite a lot of changes in one PR. The only major concern I have is the function names in the `conversions` package.

It's separated into multiple independent commits. So they can be reviewed separately while still gives reviewer the ability to peak the high-level architecture and usage.
(btw, even in this PR, you asked how a unused function will be used, submit this in one PR instead of 7 PR is to reduce such questions, 🤣 )